__tests__/p2sh/tx-proposal-mint-tokens.test.js

```diff
@@ -81,7 +81,7 @@ describe('mint-tokens tx-proposal api', () => {
       })
       .set({ 'x-wallet-id': walletId });
     expect(response.status).toBe(200);
-    expect(response.body.error).toEqual('Not enough HTR tokens for deposit: 10000000000 required, 89600 available');
+    expect(response.body.error).toContain('Not enough HTR tokens for deposit: 10000000000 required');
```

Comment:
Using the best utxo selection algorithm we don't return the amount available when we don't have enough funds. Should we create an issue to improve this in the lib?